java/core/src/test/org/apache/orc/TestNewIntegerEncoding.java

```diff
@@ -692,7 +692,8 @@ public void testPatchedBaseNegativeMin(OrcFile.EncodingStrategy encodingStrategy
 
   @ParameterizedTest
   @MethodSource("data")
-  public void testPatchedBaseNegativeMin2(OrcFile.EncodingStrategy encodingStrategy) throws Exception {
+  public void testPatchedBaseNegativeMin2(OrcFile.EncodingStrategy encodingStrategy
+                                          ) throws Exception {
```

Comment:
This PR seems to propose inconsistent changes. For example, line 649 is changed to 
```
 public void testPatchedBaseNegativeMin(OrcFile.EncodingStrategy encodingStrategy)
                                          throws Exception {
```
And, line 696 is changed to
```
public void testPatchedBaseNegativeMin2(OrcFile.EncodingStrategy encodingStrategy
                                          ) throws Exception {
```

Could you review the changes once more and make them consistent?

Reply:
Shall fix this. I remember this case. I found a similar usage in code and might have followed it. Irrespective, uniformity is needed